src/expandableCalendar/agendaList.js

```diff
@@ -163,10 +163,6 @@ class AgendaList extends Component {
     _.invoke(this.props, 'onMomentumScrollEnd', event);
   };
 
-  onScrollToIndexFailed = info => {
-    console.warn('onScrollToIndexFailed info: ', info);
```

Comment:
Please make this a default if `onScrollToIndexFailed` is not passed by the user.

Reply:
@Inbal-Tish  Done.

Reply:
Where is the else statement? `else { console.warn('onScrollToIndexFailed info: ', info); }`...

Reply:
@Inbal-Tish Thought the library shouldn't warn. That's why removed it. Have added it again.